storytower/src/apolloClient.js

```diff
@@ -111,6 +111,7 @@ query getChapterDetails($chapterIds: [ID!]!) {
     story {
       _id
     }
+    uploaded
```

Comment:
This was a new field added a while ago to keep track of when chapters are being saved to the database.